OpenKeychain/src/main/java/org/sufficientlysecure/keychain/receiver/NetworkReceiver.java

```diff
@@ -0,0 +1,54 @@
+package org.sufficientlysecure.keychain.receiver;
+
+import android.content.BroadcastReceiver;
+import android.content.ComponentName;
+import android.content.Context;
+import org.sufficientlysecure.keychain.service.KeyserverSyncAdapterService;
+import android.content.Intent;
+import android.content.pm.PackageManager;
+import android.net.ConnectivityManager;
+import android.net.NetworkInfo;
+
+import org.sufficientlysecure.keychain.Constants;
+import org.sufficientlysecure.keychain.util.Log;
+
+public class NetworkReceiver extends BroadcastReceiver {
+    private static final String ACTION_UPDATE_ALL = "update_all";
```

Comment:
Constants are used to prevent copy-paste errors and also so that changing that value of a constant doesn't require making the same change in other places it's used. Please make the constant in the service public and access it instead of creating another one here.